extensions/scarb-cairo-run/src/main.rs

```diff
@@ -34,24 +37,29 @@ struct Args {
 
 fn main() -> Result<()> {
     let args: Args = Args::parse();
+
+    let ui = Ui::new(Verbosity::default(), OutputFormat::Text);
```

Comment:
Does it take `SCARB_UI_VERBOSITY` into consideration?

Reply:
Unfortunately not, it's something I/somebody will have to add later. I'll make an issue about this, as I already forgot 😅

Reply:
https://github.com/software-mansion/scarb/issues/592